pulp_python/tests/functional/api/test_crud_content_unit.py

```diff
@@ -63,122 +64,136 @@ def test_02_upload_file_with_repo(self):
         4) ensure two and it's a content unit and a repository version
```

Comment:
I know this isn't your fault but I have no idea what this was supposed to mean, lol.  Could you reword the description?